### PR TITLE
Dolby audio and video codecs added plus updates to one specification

### PR DESCRIPTION
**Pull request by mikedo**

### CODE REVIEW

**Comment by dwsinger** [comment]
Is it clear what the difference is between dva1 and dvav (and similarly for dvh1 and dvhe) since they have exactly identical descriptive text?

**Comment by mikedo** [comment]
I have some discriminating text.  How would you like to see it executed on github?

File: CSV/sample-entries.csv [hunk 1]
```diff
@@ -20,9 +20,12 @@ dtse,"DTS Express low bit rate audio, also known as DTS LBR",Audio,DTS,0xAC
 dtsh,DTS-HD High Resolution Audio,Audio,DTS,0xAA
 dtsl,DTS-HD Master Audio,Audio,DTS,0xAB
 dtsx,DTS:X,Audio,DTS,
-dvav,AVC-based “Dolby Vision”,Video,Dolby Vision,
-dvhe,HEVC-based “Dolby Vision”,Video,Dolby Vision,
+dva1,AVC-based “Dolby Vision” derived from 'avc1',Video,Dolby Vision,
```
Comment (cconcolato) [inline CSV/sample-entries.csv]: The last value should be `dolbyvision` as this is the 'internal' link name (not user visible). Same for the other 3 lines.

File: CSV/sample-entries.csv [hunk 2]
```diff
@@ -20,9 +20,12 @@ dtse,"DTS Express low bit rate audio, also known as DTS LBR",Audio,DTS,0xAC
...
+dvav,AVC-based “Dolby Vision” derived from 'avc3',Video,Dolby Vision,
+dvh1,HEVC-based “Dolby Vision” derived from 'hvc1',Video,Dolby Vision,
+dvhe,HEVC-based “Dolby Vision” derived from 'hev1',Video,Dolby Vision,
 ec-3,Enhanced AC-3 audio,Audio,ETSI AC-3,0xA6
+ec+3,Enhanced AC-3 audio with JOC,Audio,ETSI E-AC-3+JOC,
```
Comment (cconcolato) [inline CSV/sample-entries.csv]: The last value should also match the `linkname` you used in the specification.csv: etsi-eac3+

**Comment by mikedo** [comment]
@dwsinger The 4 DV sample entry descriptions were clarified.

**Comment by dwsinger** [comment]
Merged and closing

**Pull request merged**